packages/pn-personafisica-webapp/src/components/Contacts/SpecialContacts.tsx

```diff
@@ -59,13 +41,15 @@ import {
   phoneValidationSchema,
 } from '../../utility/contacts.utility';
 import DropDownPartyMenuItem from '../Party/DropDownParty';
-import DigitalContactsCard from './DigitalContactsCard';
+import AddSpecialContactDialog from './AddSpecialContactDialog';
 import ExistingContactDialog from './ExistingContactDialog';
 import PecVerificationDialog from './PecVerificationDialog';
-import SpecialContactElem from './SpecialContactElem';
 
 type Props = {
   digitalAddresses: Array<DigitalAddress>;
+  addressType: ChannelType;
```

Comment:
call this property channelType because addressType indicates another property of address model